capi/include/hyper.h

```diff
@@ -270,6 +270,26 @@ void hyper_response_free(hyper_response *resp);
  */
 uint16_t hyper_response_status(const hyper_response *resp);
 
+/*
+ Get a pointer to the reason-phrase of this response.
+
+ This buffer is not null-terminated.
```

Comment:
So how is the buffer terminated? How can we tell where the reason-phrase ends? Is it (CR)LF?

Reply:
See two lines below.

Reply:
Sorry, I must be blind! =)